.github/workflows/deploy-production.yml

```diff
@@ -2,19 +2,13 @@ name: Production Deployment
 
 on:
   workflow_dispatch:
-    inputs:
-      version:
-        description: 'Metabase version to deploy'
```

Comment:
It was not used by us, because later if we deploy after merge to master a version is deployed based on a GH variable name